packages/client/docusaurus/docs/javascript/02-guides/03-call-and-participant-state.mdx

```diff
@@ -18,48 +24,49 @@ This approach makes it possible to access the state and be notified about change
 | `startedAt$`             | `startedAt`             | The actual start time of the current call session.                                                           |
 | `callStatsReport$`       | `callStatsReport`       | When stats gathering is enabled, this observable will emit a new value at a regular (configurable) interval. |
-| `metadata$`              | `metadata`              | The `CallResponse`, see below for more information.                                                          |
-| `callRecordingList$`     | `callRecordingList`     | The latest list of recordings performed during the call.                                                     |
 | `hasOngoingScreenShare$` | `hasOngoingScreenShare` | It will return `true` if at least one participant is sharing their screen.                                   |
+| `custom$`                | `custom`                | Custom data attached to the call.                                                                            |
 | `dominantSpeaker$`       | `dominantSpeaker`       | The participant that is the current dominant speaker of the call.                                            |
 | `ownCapabilities$`       | `ownCapabilities`       | The capabilities of the local participant.                                                                   |
-
-The `CallResponse` object contains the following information:
-
-| Name               | Description                                             |
-| ------------------ | ------------------------------------------------------- |
-| `backstage`        | It's `true` if the call is in backstage mode.           |
-| `blocked_user_ids` | The users who are blocked from this call.               |
-| `created_at`       | The time the call was created.                          |
-| `created_by`       | The creator of the call.                                |
-| `custom`           | Custom data attached to the call.                       |
-| `egress`           | Broadcasting related information.                       |
-| `ended_at`         | When the call was ended.                                |
-| `ingress`          | RTMP publishing related information.                    |
-| `recording`        | It's `true` if the call is being recorded.              |
-| `session`          | Information related to the current session of the call. |
-| `settings`         | The settings for this call.                             |
-| `starts_at`        | The scheduled start time of the call.                   |
-| `team`             | Team that the call is restricted to.                    |
-| `type`             | The type of the call.                                   |
-| `updated_at`       | When the call was updated.                              |
+| `pinnedParticipants$`    | `pinnedParticipants`    | The participants that are currently pinned.                                                                  |
```

Comment:
I propose to keep the list alphabetically ordered. It is easier for maintenance and to prevent overlooking a prop.

Reply:
Fixed.